data/chains/katana-sepolia-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "katana-sepolia-testnet",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://explorer-bokuto.katanarpc.com/api"
+    },
+    "browserUrl": "https://explorer-bokuto.katanarpc.com/"
```

Comment:
There seems to be another official browser but it doesn't work https://docs.katana.network/katana/how-to/verify-contracts-on-tatara/